### DISCUSSION TITLE
Add the author field to the versions history

### DISCUSSION THREAD

**Discussion by sagdeev** [payloadcms/payload#6276] [Feature Requests & Ideas]
Hi! I would really like to add the author field to the document versions history `/admin/collections/{slug}/{docId}/versions`. This would be very convenient, because there can be many different users 🙏

**Comment by tsemachh** [comment]
As long as the author is not part of payload it will be difficult just add Custom Versions Component and add for you

**Reply by vladvlasov256** [reply]
>the author is not part of payload

I believe all the local operations and API requests include authorization data. This data is matched with a user.
